docs/essentials/building-monorepo-projects.md

```diff
@@ -21,6 +21,44 @@ Some advantages of a monorepo approach are:
 
 Semaphore comes with out-of-box support for monorepos.
 
+## Usage examples for change_in
```

Comment:
This section should be at the end of the document here, before `See also` section.

Also, since here we are talking primarily about monorepo (change_in is just a mechanism in this context), a better option for section title would be something like `Additional examples of monorepo configuration `